index.html

```diff
@@ -48,13 +48,13 @@ <h1 class="title">COVID-19 Visualizer</h1>
       </div>
       <div class="marker-controls">
         <label class="marker-checkbox" for="markers">
-          <input id="markers" type="checkbox" checked="true" onchange="dotLayer.visible($('#markers')[0].checked).draw()"></input><span>Markers</span>
+          <input id="markers" type="checkbox" checked="true" onchange="dotLayer.visible($('#markers')[0].checked).draw()"></input><span>Dots</span>
```

Comment:
Since the word "Dots" is small, I think we should increase the font size (`#timeline .marker-controls` `font-size: 14px`, for instance).  It makes the mobile target a bit bigger.